core.php

```diff
@@ -189,6 +189,12 @@
 # Register global shutdown function
 shutdown_functions_register();
 
+# Push default language to speed calls to lang_get
+if( !defined( 'LANG_LOAD_DISABLED' ) ) {
+	require_api( 'lang_api.php' );
+	lang_push( lang_get_default() );
+}
+
```

Comment:
Did you check that this is not introducing any unwanted side effects in the core's loading sequence ? I'm guessing it does not, since plugin_api.php requires lang_api.php, but just in case.

Reply:
Moved to: #2060.

Reply:
I have checked this, including in `admin\test_langs.php`.